hack/generated/apis/microsoft.resources/v1alpha1api20200601/resourcegroup_types.go

```diff
@@ -181,6 +240,42 @@ func (spec *ResourceGroupSpec) PopulateFromARM(owner genruntime.KnownResourceRef
 // SetAzureName sets the Azure name of the resource
 func (spec *ResourceGroupSpec) SetAzureName(azureName string) { spec.AzureName = azureName }
 
+func (spec *ResourceGroupSpec) ConvertSpecTo(destination genruntime.ConvertibleSpec) error {
+	dst, ok := destination.(*ResourceGroupSpec)
+	if !ok {
+		return errors.Errorf("cannot convert ResourceGroupSpec to %T", destination)
```

Comment:
Isn't this comment backwards? You're trying to confirm %T to ResourceGroupSpec, right?

Reply:
No, I think the error is right - and matches the implementation.

We're converting from the current instance (`spec`) to the provided instance (`destination`), copying properties across.

Reply:
I see what you are saying now. What threw me was that _this particular cast_ (which is what I was expecting the error message to be about) is from `destination` to `ResourceGroupSpec`, so I was expecting a comment that read something like: `expected destination to be ResourceGroupSpec, was: %T, destination` and what you're saying is more about what the overall method is trying to do.

You can feel free to ignore this as I see what you're going for now.